### PR TITLE
BNGP-5013 Change 'Connect' to 'allocate' on page to enter BGS number

### PR DESCRIPTION
**Pull request by jd-barnard**

### CODE REVIEW

**Review by thousand-leaves — APPROVED** [review]

File: packages/webapp/src/views/developer/biodiversity-gain-site-number.html [hunk 1]
```diff
@@ -13,11 +13,10 @@ <h1 class="govuk-heading-l">
...
         </h1>
 
-        <p>This is the number which connects your development project to the land you're using as your off-site gain.</p>
-        <p>The landowner should give this to you once they’ve registered the biodiversity gain site, and agreed you can use it.</p>
-        <p class="govuk-hint">For example, BGS-010124001</p>
-
-
+        <p>This is the reference number for the land you're using for off-site gain, which you want to allocate to the development project.</p>
+        <p>The landowner will get this number once they've registered the biodiversity gain site.</p>
+        <p class="govuk-hint">For example, BGS-111 222 333</p>
```
Comment (paulsimonandrews) [inline packages/webapp/src/views/developer/biodiversity-gain-site-number.html:18]: Can we switch the example back to `BGS-010124001` please. The other example is the wrong format and misleading!

**Review by paulsimonandrews — CHANGES_REQUESTED** [review]
Quick change please for example BGS

**Review by shaundefra — APPROVED** [review]
tested locally

**Comment by shaundefra** [comment]
tested locally

**Pull request merged**